src/transactions.ts

```diff
@@ -313,16 +231,7 @@ export function actionFromRLP(sdk: SDK, rlp: Buffer): Action {
                 metadata: decoded[2]
             };
         case REPORT_DOUBLE_VOTE_ACTION_ID:
-            if (decoded.length !== 3) {
-                throw new Error(
-                    "A length of a RLP list of a reportDoubleVote action must be 3"
-                );
-            }
-            return {
-                type: "reportDoubleVote",
-                message1: decodeMessage(decoded[1]),
-                message2: decodeMessage(decoded[2])
-            };
+            return ReportDoubleVote.fromBytes(rlp);
```

Comment:
It decodes twice. How about changing `fromBytes` to `fromRLP` or `fromEncodedObject`?

Reply:
Changed the method to `fromEncodeObject`